src/cli.yaml

```diff
@@ -306,3 +306,17 @@ subcommands:
                   help: Add list of reads that were merged for each consensus read. Note that this can yield very long FASTQ name lines which cannot be handled by some tools.
 
 
+  - sequences-stats:
+      about: |
+        Tool to compute stats on sequence file (from STDIN).
```

Comment:
Thanks a lot! I think the help text here should give some detail about what exactly is computed and printed in which format.